suites/squid/rbd/tier-2_rbd_migration_external_ceph.yaml

```diff
@@ -0,0 +1,169 @@
+#===============================================================================================
+# Tier-level: 2
+# Test-Suite: tier-2_rbd_migration_external_ceph.yaml
+#
+# Cluster Configuration:
+#    cephci/conf/squid/rbd/5-node-2-clusters.yaml
+#    No of Clusters : 2
+#    Each cluster configuration
+#    5-Node cluster(RHEL-8.3 and above)
+#    3 MONS, 2 MGR, 3 OSD, 1 Client
+#     Node1 - Mon, Mgr, Installer
+#     Node2 - client
+#     Node3 - OSD, MON, MGR
+#     Node4 - OSD, MON
+#     Node5 - OSD,
+#===============================================================================================
+tests:
```

Comment:
Any reason why we are adding suites only for squid? What about other versions?

Reply:
it's 8.0 feature hence it was added in squid test suites only.